omise/payment_methods/_all.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
```

Comment:
Also, the same reason as above, it gives a bit of strange look that the classname and its filename is not reflect to each other. (cannot tell why it feels that way)

Reply:
I see this point has been replied at another comment, so I just refer to that comment here instead
> It is my convention to let us know that the file contains merely a parent/base/utility classes, not a payment method that will actually be used. The underscores make the look different and keep them at the head of a file list

if that so, wouldn't be better if (just idea) instead we make it to be abstract class and name it accordingly?
I see the convention that has been implemented here is reflect to its directory and its filename. 
`omise/payment_method/alipay` becomes `OmisePaymentMethod_Alipay`.
So just feel it's strange when we introduce that `_all.php` and `_offsite.php`. 

(it's minor so I'd like to leave it to you, just share my thought)